arkh/arkh.json

```diff
@@ -0,0 +1,61 @@
+{
+    "$schema": "../chain.schema.json",
+    "chain_name": "arkh",
+    "status": "live",
+    "network_type": "mainnet",
+    "pretty_name": "Arkhadian",
+    "chain_id": "arkh",
+    "bech32_prefix": "arkh",
+    "daemon_name": "arkhd",
+    "node_home": "$HOME/.arkhd",
+    "genesis": {
+        "https://github.com/vincadian/arkh-blockchain/blob/master/genesis/genesis.json"
+    },
+    "key_algos": [
+        "secp256k1"
+    ],
+    "slip44": 118,
+    "fees": {
+        "fee_tokens": [
+            {
+                "denom": "uarkh",
+                "fixed_min_gas_price": 0.000000001
+            }
+        ]
+    },
+    "codebase": {
+        "git_repo": "https://github.com/vincadian/arkh-blockchain",
```

Comment:
@vincadian need to know which branch/release to use.  Please cut a release for the version of the code to be used on mainnet.